libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushMessaging.kt

```diff
@@ -48,7 +49,7 @@ import com.salesforce.androidsdk.util.SalesforceSDKLogger
  * retrieval of push notification registration information from a
  * private shared preference file.
  */
-object PushMessaging {
+public object PushMessaging {
```

Comment:
Why does this need to be public?  Looks like it can be accessed from the upgrade manager without this change.

Reply:
Actually it was already public (that's the default in Kotlin) and adding public was redundant (Android Studio actually complained). But I guess we should have made that class internal.

Reply:
Yeah, I had originally written internal but I couldn't remember if there was a reason it was public.  I think it's just always been public so they kt conversion didn't change that in a minor release.